src/DetailsView/actions/details-view-action-message-creator.ts

```diff
@@ -190,11 +190,10 @@ export class DetailsViewActionMessageCreator extends DevToolActionMessageCreator
         });
     };
 
-    public startOverAssessment(event: React.MouseEvent<any>, test: VisualizationType, requirement: string): void {
+    public startOverTest(event: React.MouseEvent<any>, test: VisualizationType): void {
```

Comment:
(I guess) nitpick: should we update the name of the message we are sending? Right now is `Messages.Assessment.StartOver`